web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/SelectSecurityGroups.tsx

```diff
@@ -74,20 +77,26 @@ export const SelectSecurityGroups = ({
     }
   }
 
-  async function fetchSecurityGroups() {
+  async function fetchSecurityGroups(refresh = false) {
```

Comment:
```suggestion
  async function fetchSecurityGroups({ refresh = false } = {}) {
```

This way you don't need to document the boolean args like in Go (`fetchSecurityGroups(true /* refresh */)`). ;)